app/relationfeaturesmodel.h

```diff
@@ -0,0 +1,53 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef RELATIONFEATURESMODEL_H
+#define RELATIONFEATURESMODEL_H
+
+#include "featureslistmodel.h"
+#include "featurelayerpair.h"
```

Comment:
in general best to use forward declarations where possible for classes in the input/app or qgsquick since it speeds up development (less compilation on change)

Reply:
I tried that, however, QgsRelation and FeatureLayerPair are private members and it did not work me to do forward declaration in these cases